src/modules/commentModal.js

```diff
@@ -102,9 +103,41 @@ const commentModal = async (show) => {
         },
       }).then((res)=>{
         return res;
-      }).then(()=>{
+      }).then(async()=>{
         username.value='';
         insights.value='';
+        // ajax approach
+        var xhttp = new XMLHttpRequest();
+        xhttp.onreadystatechange = function() {
+          if (this.readyState == 4 && this.status == 200) {
+            const response = JSON.parse(this.responseText);
+            if (response.length > 0){
+              commentsContainer = document.createElement('div');
+              commentsContainer.className = 'comments-container';
+              response.forEach((data)=>{
+                  const comment = document.createElement('p');
+                  comment.className = 'comment';
+                  comment.textContent = `${data.creation_date} ${data.username} ${data.comment}`
+                  commentsContainer.appendChild(comment);
+              })
+              commentsHeading = document.createElement('h5');
+              commentsHeading.className = 'comments-heading';
+              commentsHeading.textContent = `Comments (${response.length})`;
+
```

Comment:
The implemented functionality for saving comment in real time is not full functional, there are issues that you still need to fix. Please focus on how to use form submission  to save data in real time, this could help.